### PR TITLE
Doc/api ref cleanup

### PR DESCRIPTION
**Pull request by PipKat**
Edited the PY files in the lucid folder to meet PyAnsys library documentation conventions. Also made minor edits to RST files.
@AlejandroFernandezLuces I know that the PY files in the autogen folder are automatically generated from Ansys Prime Server. However, can I edit the PY files in any of the other folders? Currently, I've kept my edits to only the PY files in the lucid folder. It seems that only they are appearing in the rendered documentation. Also, error messages should be short but complete sentences, with a subject, verb and concluding punctuation. Most error messages in this library are not complete sentences. They are generally missing a verb (usually a linking verb like "is" or "are"--and most do not have concluding punctuation (which should never be an exclamation point).

### CODE REVIEW

**Comment by AlejandroFernandezLuces** [comment]
Feel free to edit any file except from the ones in the `autogen` folder, no problem 😄 

@ninad-kamat We should move those files here https://github.com/ansys/ansys-api-meshing-prime to avoid this kind of issues.

File: src/ansys/meshing/prime/lucid/mesh_util.py [hunk 1]
```diff
@@ -10,63 +10,65 @@
...
-            Path to file to be read or imported.
-
-        append : bool
-            Set it to True to append instead of overwrite.
-
-        cad_reader_route : prime.CadReaderRoute
-            Route of CadReader.
+            Path to the file to read or import.
+        append : bool, optional
+            Whether to append to the file. The default is ``False``, in which case
```
Comment (PipKat) [inline src/ansys/meshing/prime/lucid/mesh_util.py:68]: Append to what file? Would you please enhance this description?

**Comment by ninad-kamat** [comment]
> Feel free to edit any file except from the ones in the `autogen` folder, no problem 😄
> 
> @ninad-kamat We should move those files here https://github.com/ansys/ansys-api-meshing-prime to avoid this kind of issues.

The api package should only contains the gRPC API. The `autogen` files are a key part of how PyPrimeMesh works and we change them a lot whereas the gRPC API is fairly stable. Moving them to the API package does not make sense to me. The client can actually function in-process without the api package given the correct environment. That is the reason why there are two implementations of `Communicator`. The api package is just the gRPC stubs in my opinion. The `autogen` folder is far more than that. It is the very heart of PyPrimeMesh.

**Comment by PipKat** [comment]
@AlejandroFernandezLuces All PY files have been edited. In a separate PR, the tester for this library should rewrite strings as short, complete sentences with concluding punctuation.

**Comment by PipKat** [comment]
@AlejandroFernandezLuces and @ninad-kamat I'm updating the branch now. When it finishes, would one of you please approve and merge this PR? The two open comments can always be addressed later in another PR.

**Comment by AlejandroFernandezLuces** [comment]
I opened a PR to fix the comment at another time: https://github.com/ansys/pyprimemesh/issues/525

I'll give a quick review and I'll approve 🙂

**Review by AlejandroFernandezLuces — APPROVED** [review]
LGTM, merging!

**Pull request merged**